modules/react/collection/lib/useListRenderItem.tsx

```diff
@@ -7,14 +7,14 @@ import {useBaseListModel} from './useBaseListModel';
 export function useListRenderItems<T>(
   model: ReturnType<typeof useBaseListModel>,
   children: ((item: Generic, index: number) => React.ReactNode) | React.ReactNode
-) {
+): React.ReactNode {
   const items =
     typeof children === 'function'
       ? model.state.isVirtualized
         ? model.state.UNSTABLE_virtual.virtualItems.map(virtual => {
             const item = model.state.items[virtual.index];
             const child = children(item.value, virtual.index);
-            return React.cloneElement(child, {
+            return React.cloneElement(child as React.ReactElement, {
```

Comment:
I'm curious about this change. Casting like this could cause issues where we're making an assumption about what `child` could be and ignoring when `child` is not a `React.ReactElement`.